packages/curriculum-ast-validators/index.js

```diff
@@ -65,6 +65,9 @@ module.exports = {
           Object.values(questionValidator).forEach((validator) => {
             validator(ast);
           });
+      case contentTypes.EXERCISE:
+      case contentTypes.GLOSSARY:
+        return () => true;
```

Comment:
`exercises` are deprecated so dont bother
`glossary` has some validation done in the json compiler, that i could copy over, but for now don't do anything